tox.ini

```diff
@@ -39,6 +39,8 @@ description = run static analysis and style check using flake8
 commands = python -m flake8 --show-source tox setup.py {posargs}
            python -m flake8 --show-source doc tests {posargs}
            pre-commit run --all-files
+           echo "hint: run {envdir}/bint/pre-commit install" to add checks as pre-commit hook
```

Comment:
`s/bint/bin/g`

You could also just run this directly: `pre-commit install --overwrite`: [here's an example](https://github.com/pre-commit/pre-commit-hooks/blob/efdceb4e40cda10780f4646ec944f55b5786190d/tox.ini#L18)